src/systems/ec/implementations/vanilla/packages/ec-component-form-label/ec-component-form-label-print.scss

```diff
@@ -0,0 +1,34 @@
+/*
+ * Form label
+ * @define form-label
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+@mixin ecl-form-label-print() {
+  .ecl-form-label {
+    color: $ecl-color-black-100;
+    font: $ecl-font-print-m;
+    font-weight: $ecl-font-weight-print-bold;
+    margin: 0;
+  }
+
+  .ecl-form-label--hidden {
```

Comment:
do we really want to hide a label in print?

Reply:
yes, we do :)